apps/api/src/modules/job/application/offer/offer.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common'
+import { PrismaModule } from 'src/database/prisma.module'
+
+import { ApplicationModule } from '../application.module'
+import { OfferController } from './offer.controller'
+import { OfferRepository } from './offer.repository'
+import { OfferService } from './offer.service'
+
+@Module({
+  imports: [ApplicationModule, PrismaModule],
```

Comment:
Why import Application module

Reply:
it is used to get application for the job offer

Reply:
Ah I read it wrong I thought it a AppModule